fog/distribution/tests/bootstrap.rs

```diff
@@ -0,0 +1,73 @@
+use std::{
+    env::{args, set_current_dir},
+    path::PathBuf,
+    process::Command,
+};
+
+// Test that fog distro can find the spendable tx outs from the bootstrap
+#[test]
+fn test_find_spendable_tx_outs() {
```

Comment:
This is a pretty funky test. Is it generating all the keys and stuff in the target directory?
It should create a temporary directory that gets cleaned up after the test is done, and not rely on deleting stuff that might happen to exist in the current directory.

Reply:
Ok we can do that

Reply:
I added this test to try to reduce my iteration times in CD

Reply:
At the very least it is worth explaining the purpose of this.